build/templates/_attributes.py.mako

```diff
@@ -104,6 +104,29 @@ class AttributeEnum(object):
         return self._underlying_attribute.__set__(session, value.value)
 
 
+class AttributeBooleanEnum(object):
```

Comment:
note the missing newline compared to other classes

Reply:
The missing newline has been added in my latest commit